VaderSharp/VaderSharp/Extensions.cs

```diff
@@ -24,7 +24,7 @@ public static bool IsUpper(this string word)
         /// <returns></returns>
         public static string RemovePunctuation(this string word)
         {
-            return new string(word.Where(c => !char.IsPunctuation(c)).ToArray());
+            return System.Text.RegularExpressions.Regex.Replace(word, "[^\\s\\w]", "");
```

Comment:
@Chr1st1anSzech hey, just did some profiling with this - the regex is actually twice as slow. 
`            const string testSentence = "This is a sample, test sentence, which has some punctuation!";

            Stopwatch sw = new Stopwatch();

            sw.Start();

            for (int i = 0; i < 100000; i++)
            {
                var test = new string(testSentence.Where(c => !char.IsPunctuation(c)).ToArray());
            }

            sw.Stop();

            Console.WriteLine("Elapsed={0}", sw.Elapsed);

            sw.Reset();
            sw.Start();


            for (int i = 0; i < 100000; i++)
            {
                var test = System.Text.RegularExpressions.Regex.Replace(testSentence, "[^\\s\\w]", "");
            }

            sw.Stop();

            Console.WriteLine("Elapsed={0}", sw.Elapsed);
            Console.ReadKey();`